### PR TITLE
ztests CLI support for -h and -v

### PR DESCRIPTION
**Pull request by pantuza**
This pull request adds compatibility for **ztests** with other binaries from zmap compilation. 
It is a reference to the issue #404 

**ztests** binary now has the following command line options:

* ztests -h
* ztests --help
* ztests -V
* ztests --version

Regards.

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -177,6 +178,11 @@ add_custom_command(OUTPUT zitopt.h
     DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/zitopt.ggo"
 )
 
+add_custom_command(OUTPUT ztopt.h
+	COMMAND gengetopt -C --no-help --no-version --unamed-opts=SUBNETS -i "${CMAKE_CURRENT_BINARY_DIR}/ztopt.ggo" -F "${CMAKE_CURRENT_BINARY_DIR}/ztopt"
```
Comment (dadrian) [inline src/CMakeLists.txt]: `--unamed-opts` should probably be excluded, since I don't think ztest takes arguments.

**Comment by zakird** [comment]
This looks fine to me.

File: src/ztopt.ggo.in [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+purpose "A tests harness tool for zmap"
+
+section "Additional options"
+
+option "help"                   h "Print help and exit"
+    optional
+option "version"                V "Print version and exit"
+    optional
+
+text "\nExamples:\n\
```
Comment (dadrian) [inline src/ztopt.ggo.in]: Not sure if this text is necessary.

**Comment by pantuza** [comment]
@dadrian and @zakird, thank you for the code review. I totally agree with the changes.

**Comment by dadrian** [comment]
Thank you for your contribution!

**Pull request merged**